### PR TITLE
Doc/taipy#1748 - Stop supporting python 3.8

### LINKED ISSUE #1748: Stop support for Python 3.8
Stop supporting version 3.8 of Python.

### PR DESCRIPTION
**Pull request by trgiangdo**
Resolves https://github.com/Avaiga/taipy/issues/1748

### CODE REVIEW

**Review by jrobinAV — APPROVED** [review]

File: docs/tutorials/articles/understanding_gui/index.md [hunk 1]
```diff
@@ -37,13 +37,13 @@ accordingly.
...
 
-Ensure you have Python 3.8 or newer, then install Taipy:
+Ensure you have Python 3.9 or newer, then install Taipy:
 
 ```bash
 pip install taipy
 ```
 
-For the NLP feature (Python 3.8 to 3.10), install:
+For the NLP feature (Python 3.9 to 3.10), install:
```
Comment (FabienLelaquais) [inline docs/tutorials/articles/understanding_gui/index.md:46]: Is this still true? With Python 3.12 out?

File: docs/userman/run-deploy/deploy/linux/redhat.md [hunk 2]
```diff
@@ -67,11 +67,11 @@ taipy
 
 On your local machine, start the application by doing:
 ```console
-$ python3.8 -m pip install -r requirements.txt
+$ python3.9 -m pip install -r requirements.txt
```
Comment (FabienLelaquais) [inline docs/userman/run-deploy/deploy/linux/redhat.md:70]: Why not just `python3` or even `python`?? Why bother?

**Review by FabienLelaquais — APPROVED** [review]
Nice!
Please make sure that Marketing (Rym) is aware of the move so the web site is up-to-date as well.

File: docs/tutorials/articles/understanding_gui/index.md (hunk 1 above)
Comment (trgiangdo) [reply docs/tutorials/articles/understanding_gui/index.md:46]: It's for PyTorch. I'm not sure if it remains true till this day. Let me check

File: docs/userman/run-deploy/deploy/linux/redhat.md (hunk 2 above)
Comment (trgiangdo) [reply docs/userman/run-deploy/deploy/linux/redhat.md:70]: It's on the deployment doc for Redhat. We've already had a `info` block saying that you can or should replace `python3.9` with `python`

**Pull request merged**